packages/core/src/amazonq/webview/ui/tabs/constants.ts

```diff
@@ -15,6 +15,8 @@ export type TabTypeData = {
 
 export const createPromptCommand = 'Create a new prompt'
 
+export const promptFileExtension = '.prompt'
```

Comment:
This file is defined and lives in the compiled js file of the webview.

If you want this variable be referenced by IDE code, you should move this out to somewhere else.